amy/autoemails/actions.py

```diff
@@ -1175,3 +1181,9 @@ def check(term: Term):
 
     def get_additional_context(self, objects, *args, **kwargs):
         return dict()
+
+    def get_merge_data(self) -> Optional[Dict]:
+        # Returning an empty dictionary is necessary
+        # if you want to send an individual email to each recipient
+        # but have no per-user configuration
+        return {}
```

Comment:
We may need to add user details here. Did you ask @fmichonneau and others how the email message should look like? Maybe they'd like to add user name to the email body?

Reply:
I don't think I ever received any copy. @fmichonneau do you have a sense of what you would like the email to look like for consents? I just have a bare-bones message.

Reply:
Let's talk about it when we meet. I'll draft something after that. Thanks!